CMakeLists.txt

```diff
@@ -1,6 +1,8 @@
 cmake_minimum_required( VERSION 2.8 FATAL_ERROR )
 project (urdfdom_headers)
 
+include(GNUInstallDirs)
```

Comment:
GNUInstallDirs does not play a different game in Windows, there is no logic in its cmake code about Windows or Mac so it will generate the same scheme than in Linux.

Note that [there is logic in the current code](https://github.com/ros/urdfdom_headers/blob/master/CMakeLists.txt#L24) that controls the destination in Windows. This PR should only modify UNIX systems behaviour.